src/renderer/components/swap/Swap.styles.tsx

```diff
@@ -4,6 +4,8 @@ import { palette } from 'styled-theme'
 
 import { media } from '../../helpers/styleHelper'
 import { AssetInput as AssetInputBase } from '../uielements/assets/assetInput'
+import { Drag as BaseDrag } from '../uielements/drag'
+import { Label as UILabel } from '../uielements/label'
```

Comment:
[nitpick] Please use same name convention for elements imported from `uielements/`. In other `*.styles.tsx` files you will find something like `import { Comp as UIComp } from '../uielements/comp''